types/knockout/test/index.ts

```diff
@@ -173,7 +173,7 @@ function test_bindings() {
 
     ko.bindingHandlers.yourBindingName = {
         init: function (element, valueAccessor, allBindingsAccessor, viewModel, bindingContext) {
-        	return { "controlsDescendantBindings": true };
+            return { "controlsDescendantBindings": true };
```

Comment:
Are these indentation changes needed?

Reply:
Needed is a strong word. They were generated by vs code auto formatted and reviewed by me as **useful**. Original indentation was non-standard (didn't respect the rest of the file indentation).